requirements-minimal.txt

```diff
@@ -6,8 +6,9 @@ h5py>=3.9.0
 hdmf>=3.11.0
 hdmf_zarr>=0.4.0
 pynwb>=2.3.2;python_version>='3.8'
+nwbinspector>=0.4.31
```

Comment:
So looking at the dependencies of the inspector this still does not solve the problem in #651. The inspector does have the following dependenies that mainly relate to download and uploading data.

https://github.com/NeurodataWithoutBorders/nwbinspector/blob/0e3640048f18838762af98ccbf071b07f0b39e48/requirements.txt#L2-L5